app/templates/admin/cceMinor.html

```diff
@@ -60,70 +60,146 @@ <h2>CCE Minor Progress</h2>
 <br> <br>
 <div class="row">
     <div class="col">
-        <h2>Interested Students</h2>
+        <h2>CCE Minor Candidates</h2>
     </div>
-    <div class="col-auto">
-        <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addInterestedStudentsModal" style="margin:5px;">Add Students</a>
-        <button class="btn btn-primary" id="emailAllInterested">Email All</button>
+    <div class="col-auto d-flex align-items-center">
+      <a class="btn btn-success m-1" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addInterestedStudentsModal">Add Students</a>
+      <div class="dropdown">
+        <button class="btn btn-primary dropdown-toggle" type="button" id="emailAllInterestedDropdown" data-bs-toggle="dropdown" aria-expanded="false">
```

Comment:
Don't love this being called email all now that we can choose to not email all students. Maybe just email?